src/js/media-time-display.js

```diff
@@ -141,6 +142,9 @@ class MediaTimeDisplay extends MediaTextDisplay {
   }
 
   toggleTimeDisplay() {
+    if (this.hasAttribute('notoggle')) {
```

Comment:
nitpick(blocking): use aforementioned `Attributes.NO_TOGGLE` here for consistency

Reply:
should we use the prop instead?